src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,106 @@
+package duke.task;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private ArrayList<Task> taskList;
+
+    public TaskList(ArrayList<Task> taskList) {
+        this.taskList = taskList;
+    }
+
+    public void add(Task newTask) {
+        this.taskList.add(newTask);
+        System.out.println("    Got it. I've added this task:");
```

Comment:
Could this be extracted out to your Ui class, considering it is dealing with printing out text?